apps/balancer-tools/src/app/apr/(utils)/getFilteredApiUrl.ts

```diff
@@ -67,9 +70,36 @@ export default function getFilteredDateApiUrl(
     .map(([key, value]) => `${key}=${value}`)
     .join("&");
 
-  return `${BASE_URL}/apr/api?startAt=${formatDateToMMDDYYYY(
-    startAt,
-  )}&endAt=${formatDateToMMDDYYYY(endAt)}${
+  return `startAt=${formatDateToMMDDYYYY(startAt)}&endAt=${formatDateToMMDDYYYY(
+    endAt,
+  )}&${params}`;
+}
+
+export function generateApiUrlWithParams(
+  startAt: Date,
+  endAt: Date,
+  searchParams?: SearchParams | null,
+  poolId?: string,
+) {
+  const queryParams = generateQueryParams(startAt, endAt, searchParams);
+  return `${BASE_URL}/apr/api?${queryParams}${
     poolId ? `&poolId=${poolId}` : ""
-  }&${params}`;
+  }`;
+}
+
+export function generateRedirectUrlWithParams(
+  startAt: Date,
+  endAt: Date,
```

Comment:
shouldn't we just default to using a MMDDYYYY string instead of everywhere instead of passing it to date then passing it again back to this format? I notice we've done it a bunch of times across the code. Part of me also thinks using a timestamp would've made the whole implementation easier (although, admittedly, the links less readable)